pkg/operation/botanist/component/kubeapiserverexposure/kube_apiserver_sni.go

```diff
@@ -219,6 +253,84 @@ func (s *sni) emptyVirtualService() *istionetworkingv1beta1.VirtualService {
 	return &istionetworkingv1beta1.VirtualService{ObjectMeta: metav1.ObjectMeta{Name: v1beta1constants.DeploymentNameKubeAPIServer, Namespace: s.namespace}}
 }
 
+func (s *sni) emptyAuthorizationPolicy(name string) *istiosecurity1beta1.AuthorizationPolicy {
+	return &istiosecurity1beta1.AuthorizationPolicy{ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: s.values.IstioIngressGateway.Namespace}}
+}
+
+func (s *sni) emptyAuthorizationPolicyApiServer() *istiosecurity1beta1.AuthorizationPolicy {
+	return s.emptyAuthorizationPolicy(s.namespace + "-api-server")
+}
+
+func (s *sni) emptyAuthorizationPolicyVpnServer() *istiosecurity1beta1.AuthorizationPolicy {
+	return s.emptyAuthorizationPolicy(s.namespace + "-vpn-server")
+}
+
+func (s *sni) getAccessControlSpec() (*istioapisecurityv1beta1.AuthorizationPolicy, error) {
+	var err error
+	action := istioapisecurityv1beta1.AuthorizationPolicy_ALLOW
```

Comment:
Note that the default action in AuthorizationPolicies is `ALLOW` so you could skip specifying it explicitly here.

Reply:
If you want I can remove, but I think I like this to be explicit as it conveys the intention and also if Istio's default changes the implementation does not break.

Reply:
I don't really have a strong opinion. But yeah let's keep it so it's a bit more clear.